### PR TITLE
Add column to Snowflake DS - #1 Create Snowflake connection

### PR DESCRIPTION
**Pull request by rmorehig**
Related #38 

This PR adds the following:
- Init data project
- Create snowflake datasource

### CODE REVIEW

**Comment by alrocar** [comment]
Looking at CI it seems it didn't deploy anything because the Data Source already exists:

```
** Detecting changes from last release ...
** Diffs from current release '41170973016cd3890d2b8774a8730ba39c2bb60e' and new '007514765d89e31f82c2b034115dfd6e00d6a107':
	no diffs detected
```

**Comment by rmorehig** [comment]
@alrocar I had to recreate the workspace so it failed the first time but I created a fresh new one

File: add_column_snowflake_ds/deploy/0.0.1/ci-deploy.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+set -e
+
+tb deploy --populate --fixtures --wait
```
Comment (alrocar) [inline add_column_snowflake_ds/deploy/0.0.1/ci-deploy.sh:4]: both ci-deploy and cd-deploy are the default ones, so they are not needed, can you try to remove them? (also .tinyenv is probably not necessary to bump version)

Comment (rmorehig) [reply add_column_snowflake_ds/deploy/0.0.1/ci-deploy.sh]: Removed

**Review by alrocar — APPROVED** [review]
🚀

**Pull request merged**